src/apps/appstore/i18n/en-US.json

```diff
@@ -88,6 +88,10 @@
 		},
 		"dangerDisableDialog": {
 			"content": "This will disable the Marketplace connector. The next time your KAZOO nodes start, apps purchased through the Marketplace will not be started. Do not forget to also remove those applications from your Kazoo nodes start up list."
+		},
+		"unlinkedCluster": {
+			"mainText": "Hey, if you unliked your cluster using the marketplace portal",
```

Comment:
Change this to a better help text, ask Emily to help. Something along:

```
Hey, we unable to get your cluster information from Marketplace. It seems you unlinked your cluster using Marketplace Portal, if that is the case please click the button below to clear linkage status from cluster. You may want to re-link your cluster after.

If that is not the case check your Cluster ID is not changed and Cluster Secret is set and correct them.

If you think you Cluster ID or Secret is compromise please check the Marketplace place immediately and contact us for help
```

Run that with Emily first please.

Reply:
updated with emily's text